getdokan/dokan
# Enhancement/admin order details page

### All Submissions:






### Changes proposed in this Pull Request:

![image](https://github.com/getdokan/dokan/assets/32583103/da0714f6-a42e-4470-813c-099939559cd8)

### Closes 

* Closes https://github.com/getdokan/dokan-pro/issues/2676


### How to test the changes in this Pull Request:

* Order, complete order, refund, discount add coupon and see commission information and related order.


### Changelog entry

*Add commission information and related order information on orders and child orders.*


### Before Changes

Describe the issue before changes with screenshots(s).


### After Changes

Describe the issue after changes with screenshot(s).


### Feature Video (optional)

Link of detailed video if this PR is for a feature.


### PR Self Review Checklist:

* Code is not following code style guidelines
* Bad naming: make sure you would understand your code if you read it a few months from now.
* KISS: Keep it simple, Sweetie (not stupid!).
* DRY: Don't Repeat Yourself.
* Code that is not readable: too many nested 'if's are a bad sign.
* Performance issues
* Complicated constructions that need refactoring or comments: code should almost always be self-explanatory.
* Grammar errors.


### FOR PR REVIEWER ONLY:

> As a reviewer, your feedback should be focused on the idea, not the person. Seek to understand, be respectful, and focus on constructive dialog.

> As a contributor, your responsibility is to learn from suggestions and iterate your pull request should it be needed based on feedback. Seek to collaborate and produce the best possible contribution to the greater whole.
